hatchways/team-bluejay
# Kareem chef profile page #31

Ticket #31 

Description:
Created a chef profile page and the necessary components 

Testing:
- checkout the branch 
- create a stub data const in the chef profile and a list of meal items too
- make the /profile route return chef profile page (this is temporary for testing purpose)
- once you can see the page, verify that it is visually correct

Notes:
there are some pending tasks 
- Edit chef profile button needs to be added 
- Real data needs to be plugged in from the backend into the Front-end
- in the Profile page, we need to check if logged in user is_chef === true and if it is true then we return <ChefProfile chef={user} />  instead of <Profile />
- meal.tagline should be changed and integrated with meal.servings in the backend
